Assets/Plugins/Source/Editor/Utility/PackageFileUtility.cs

```diff
@@ -208,20 +248,22 @@ private static IEnumerable<FileInfoMatchingResult> FindMatchingFiles(string root
             CancellationToken cancellationToken = default,
             Action<string> postProcessCallback = null)
         {
-            // TODO: A major improvement to this system would be to automatically include .meta files, and gracefully
-            //       handle instances where the meta file *should* exist, but does not - gracefully in this context
-            //       would mean that if the .meta file exists, it should just be copied over; whereas if there is a
-            //       file entry copied over that does not have a corresponding meta file in the source, it indicates
-            //       that a meta file entry is MISSING from the source, and that should stop the creation of a package.
-
-
             Directory.CreateDirectory(destination);
 
             GetFileSystemOperations(
                 destination, 
                 matchingResults,
                 out List<FileUtility.CopyFileOperation> copyOperations);
 
+            if (0 == copyOperations.Count)
+            {
+                Debug.LogWarning("There were no files that need to be moved to create the package (nothing seems to have changed).");
```

Comment:
If there are no things to copy, maybe we can exit out here instead of continuing to the await?

Reply:
Because a function that is defined with the `async` keyword _must_ return a type of `Task`, we could not just stop here, we would need to return an empty `Task`, which I thought would be more confusing that letting the `await` keyword accomplish that behavior for us.

Reply:
Marking this as resolved because these changes actually got merged in the other PR, and are no longer a part of _this_ PR.